### ISSUE TITLE
Something changed with the configuration values / bundle?

### ISSUE THREAD

**Issue by sboehle** [supermerill/SuperSlicer#4098]
### What happened?

With the latest nightly build, when i start it with a configuration set from the previous build, i get a message that the configuration bundle is loaded but that there are configuration values that were not recognized:
![image](https://github.com/supermerill/SuperSlicer/assets/60656814/7629aa03-8160-4429-a286-9dac7a310ad9)

List of settings:

print settings:
Unknow setting: complete_objects_one_brim
(value: "0")
Unknow setting: threads
(value: "16")
Unknow setting: wall_add_middle_threshold
(value: "75%")
Unknow setting: wall_split_middle_threshold
(value: "50%")

filament settings:
Unknow setting: cooling
(value: "1")
Unknow setting: fan_always_on
(value: "1")

I didn't found any corresponding fields or switches in the GUI.

Probably it has something to do with the improvements regarding "better import from prusa/orca" or these configuration values are obsolete and simply cleaned up.

When i open an project file (.3mf) from Superslicer, i get the same info regarding not recognized configuration values.

I don't know if this is a bug or by-design. 

### Project file & How to reproduce

Simply open the latest nightly build with an "old" configuration folder in the same root like SuperSlicer.

### Version

latest nightly build from January 30th

### Operating system

win 11

### Printer model

Voron 2.4

**Comment by supermerill** [comment]
yes, I added more messages about what values are not converted, when reading a configuration (to have better result with prusa & orca import)
But this also pop for me, currently looking at it.
